woocommerce/woocommerce
# New CFE workflow

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This workflow gets triggered when the CFE PR is merged into the release branch, cherry-picking the commit and creating a new PR against trunk.







### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

I am sure there must be a better way to test this, but these are the steps I followed:

Please disable some common workflows in your forked repo, like CI, etc, to save time.

1. Fork this repo in a different org, make sure the name is `woocommerce`.
2. Add this as a remote in your local.
3. Create a new branch `release/9.4` out of this branch and push that upstream in your new repo.
4. Create a regular fix - PR with changelog out of `release/9.4` as you usually do for `trunk`.
5. Squash and merge the PR in release branch.
6. It should trigger the new cherry-pick workflow, that will create a PR to merge the change in the `trunk` like [this](https://github.com/Alconomy/woocommerce/pull/6).
7. Note that workflow will fail due to slack workflow, but it shall work for `woocommerce` repo.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
